datadog/resource_datadog_powerpack.go

```diff
@@ -602,6 +616,51 @@ func buildDatadogPowerpack(ctx context.Context, d *schema.ResourceData) (*datado
 
 }
 
+func normalizeDashboardWidgetDef(widgetDef map[string]interface{}) (map[string]interface{}, diag.Diagnostics) {
+	var diags diag.Diagnostics
+	// Dashboard widgets set live span at the widget level, we don't allow that for powerpack widgets
+	// where live span is set at the resource level.
+	if widgetDef["live_span"] != nil {
+		diags = append(diags, diag.Diagnostic{
+			Severity: diag.Error,
+			Summary:  fmt.Sprintf("live_span must be set at the powerpack level and will be applied to all widgets"),
```

Comment:
**nitpick(non-blocking):** Language tweak for consideration /cc @DataDog/documentation folks
```suggestion
			Summary:  fmt.Sprintf("live_span must be set for all powerpack resources (to be applied to all widgets within the powerpack)"),
```

Reply:
@woodb  good idea - i'll make that adjustment.